client/src/styles/ProductCard.css

```diff
@@ -128,4 +129,54 @@
 .product__icon {
 	width: 18pt;
 	margin: 4pt auto 2pt;
+}
+
+@media (min-width: 675pt) {
+	.product__card {
+		max-width: 360pt;
+		min-width: 300pt;
+		display: flex;
+		flex-direction: column;
+		justify-content: space-around;
+		align-items: center;
+		align-content: center;
+		margin: 4pt auto;
+	}
+
+	.product__title {
+		margin: 14pt 12pt 15pt;
+		padding: 6pt 6pt 15pt;
+		height: 50pt;
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		position: relative;
+	}
+
+	.product__avail-container {
+		background-color: #E5E5E5;
+		border-radius: 6pt;
+		margin-left: 12pt;
+		margin-right: 12pt;
+		bottom: -15px;
+		width: 280pt;
+		height: 37.5pt;
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		position: absolute;
+	}
+
+	.product__avail-text {
+		white-space: pre-line;
+		font-size: medium;
+	}
```

Comment:
Please remove (redundant)

Reply:
Thank you for pointing it out! I removed those.